LOCALAPP.md

```diff
@@ -12,8 +12,8 @@ These are some of the required steps, tips and tricks when it comes to running a
 2. Newest [Git](https://git-scm.com/downloads)
 3. A code editor - we like [Visual Studio Code](https://code.visualstudio.com/Download)
     - Also install [recommended extensions](https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions) (f.ex. [C#](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp))
-4. For Windows/MacOS [Docker Desktop](https://www.docker.com/products/docker-desktop)
-5. Update hosts file (C:/Windows/System32/drivers/etc/hosts) by adding the following values. On MacOS add the same values to values /private/etc/hosts using cmd `sudo nano /private/etc/hosts`.
+4. [Docker Desktop](https://www.docker.com/products/docker-desktop) for Windows/MacOS, or install Docker from your package manager on Linux
```

Comment:
News to me! :birthday: After noticing that installation instructions for my distribution was 'download the latest file and run it', I also noticed [it runs containers inside a virtual machine](https://docs.docker.com/desktop/linux/install/#why-docker-desktop-for-linux-runs-a-vm) instead of running them natively. That kind-of killed my excitement for it. As production goes the opposite direction by running containers in kubernetes, I'm guessing we should decouple from the container runtime more, not tie into Docker Desktop as the only possible solution. On my first day I tried running this all in Podman instead (unsuccessfully), which is more closely related to kubernetes.

Anyway, I'll reword it to make Docker Desktop an option on Linux too! Might be a preferable for Linux users who aren't as comfortable with the CLI.

Reply:
> I also noticed [it runs containers inside a virtual machine instead of running them natively.

Interesting... Haven't tested it myself. I understand the reasoning.

> Might be a preferable for Linux users who aren't as comfortable with the CLI.

Yup, simple for the common people.

> As production goes the opposite direction by running containers in kubernetes

Yeah. I hope we might get to a place where we can run a complete Platform locally on local Kubernetes (and [on a Pi](https://github.com/Altinn/altinn-studio/issues/5912) 😁), and for that to be easy for everybody, Docker Desktop might be [a possible solution](https://docs.docker.com/desktop/kubernetes/) also for that.